conf/supervisord-dev.conf

```diff
@@ -1,8 +1,14 @@
 [supervisord]
 nodaemon = true
-environment=PYTHONUNBUFFERED="1"
```

Comment:
I'm not sure what this was doing here or whether it's necessary. Other projects don't have it. I think probably just let the cookiecutter remove this.